cmd/wof_extract_sqlite.js

```diff
@@ -9,7 +9,9 @@ const combinedStream = require('combined-stream');
 
 const SQLITE_REGEX = /whosonfirst-data-[a-z0-9-]+\.db$/;
 
-const WOF_DIR = process.env.WOF_DIR || '/data/whosonfirst-data/sqlite';
+const WOF_DIR = process.env.WOF_DIR || // TODO: deprecate WOF_DIR env var after some time
+  config.whosonfirst.datapath || // Preferred method of finding WOF data is to autodetect from pelias.json
```

Comment:
With ES6 this can be written as `config?.whosonfirst?.datapath` which will avoid fatal errors if the parent path doesn't exist.

Likely not an issue in practise as we have [a default value](https://github.com/pelias/config/blob/master/config/defaults.json#L117)